packages/headless/src/features/triggers/triggers-state.ts

```diff
@@ -48,6 +48,15 @@ export interface TriggerState {
    * The notifications to display to the user after receiving notification triggers.
    */
   notifications: string[];
+
+  /**
+   * The modification that should be applied as a result of a query trigger.
+   */
+  queryModification: {
+    originalQuery: string;
+    modification: string;
+    ignore: string;
```

Comment:
The name `ignore` makes me expect this to be a boolean, but it's a string. I think a name like `queryToIgnore` would be more clear.

Also, I think that `ignore` should be optional, since an empty string is also technically a query.

Reply:
I think each three properties here should get a jsdoc description.